types/jasmine/index.d.ts

```diff
@@ -436,58 +483,66 @@ declare namespace jasmine {
-        xit(desc: string, func: () => void): XSpec;
-        addCustomEqualityTester(equalityTester: CustomEqualityTester): void;
-        addMatchers(matchers: CustomMatcherFactories): void;
-        specFilter(spec: Spec): boolean;
+        provideFallbackReporter(reporter: CustomReporter): void;
         /**
-         * @deprecated Use oneFailurePerSpec option in {@link jasmine.Env.configure} instead.
+         * @deprecated Use random option in {@link jasmine.Env.configure} instead.
          */
-        throwOnExpectationFailure(value: boolean): void;
+        randomTests(): boolean;
         /**
-         * @deprecated Use failFast option in {@link jasmine.Env.configure} instead.
+         * @deprecated Check random option in {@link jasmine.Env.configuration} instead.
          */
-        stopOnSpecFailure(value: boolean): void;
+        randomizeTests(value: boolean): void;
         /**
          * @deprecated Use seed option in {@link jasmine.Env.configure} instead.
          */
-        seed(seed: string | number): string | number;
-
+        seed(value?: number | string): boolean;
         /**
          * Sets a user-defined property that will be provided to reporters as
          * part of the properties field of SpecResult.
          * @since 3.6.0
          */
-        setSpecProperty(key: string, value: unknown): void;
-
+        setSpecProperty: typeof setSpecProperty;
         /**
          * Sets a user-defined property that will be provided to reporters as
          * part of the properties field of SuiteResult.
          * @since 3.6.0
          */
-        setSuiteProperty(key: string, value: unknown): void;
-
-        provideFallbackReporter(reporter: CustomReporter): void;
-        throwingExpectationFailures(): boolean;
-        allowRespy(allow: boolean): void;
-        randomTests(): boolean;
+        setSuiteProperty: typeof setSuiteProperty;
         /**
-         * @deprecated Use random option in {@link jasmine.Env.configure} instead.
+         * @deprecated Use specFilter option in {@link jasmine.Env.configure} instead.
          */
-        randomizeTests(b: boolean): void;
-        clearReporters(): void;
-        configure(configuration: EnvConfiguration): void;
+        specFilter(spec: Spec): boolean;
+        /**
+         * @deprecated Use failFast option in {@link jasmine.Env.configure} instead.
+         */
+        stopOnSpecFailure(value: boolean): void;
+        /**
+         * @deprecated Check failFast option in {@link jasmine.Env.configuration} instead.
+         */
+        stoppingOnSpecFailure(): boolean;
+        /**
+         * @deprecated Use oneFailurePerSpec option in {@link jasmine.Env.configure} instead.
+         */
+        throwOnExpectationFailure(value: boolean): void;
+        /**
+         * @deprecated Check oneFailurePerSpec option in {@link jasmine.Env.configuration} instead.
+         */
+        throwingExpectationFailures(): boolean;
+        topSuite(): Suite;
```

Comment:
This isn't in the docs either.

Reply:
https://github.com/jasmine/jasmine/commit/ce850c472a69b59b10abb24d93b6955e864b078a#diff-46cdc2a30e1879f73f5a62a709840911600e79ff44702785f1fafb4234789c57R1628

same

Reply:
The docs were added recently so they are in [edge](https://jasmine.github.io/api/edge/Env.html)